genesis.json

```diff
@@ -0,0 +1,32 @@
+{
+  "genesis": {
+    "config": {
+      "chainId": 420,
+      "homesteadBlock": 0,
+      "eip150Block": 0,
+      "eip150Hash": "0x0000000000000000000000000000000000000000000000000000000000000000",
+      "eip155Block": 0,
+      "eip158Block": 0,
+      "byzantiumBlock": 0,
+      "constantinopleBlock": 0,
+      "petersburgBlock": 0,
+      "istanbulBlock": 0,
+      "clique": {
+        "period": 15,
+        "epoch": 30000
+      }
+    },
+    "nonce": "0x0",
+    "timestamp": "0x61bc0b0a",
+    "extraData": "0x0000000000000000000000000000000000000000000000000000000000000000cf281c8a32fb8d1a3bb4a64055c85cf20a1ed33e0000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000",
```

Comment:
I think that it's necessary to add 0xcf281c8a32fb8d1a3bb4a64055c85cf20a1ed33e's genesis balance and the address keystore file, so we can use init node directly and use the addr send tx.

Reply:
Good catch, I will add it!